website/src/_includes/footer.html

```diff
@@ -42,11 +42,12 @@
                                                                                                                                 width="14" height="14"
                                                                                                                                 alt="External link."></a></div>
         <div class="footer__cols__col__link"><a href="{{'/contribute/presentation-materials/'|prepend:site.baseurl}}">Media</a></div>
+       <div class="footer__cols__col__link"><a href="{{'/community/in-person/'|prepend:site.baseurl}}">Events/Meetups</a></div>
       </div>
       <div class="footer__cols__col footer__cols__col--md">
         <div class="footer__cols__col__title">Resources</div>
         <div class="footer__cols__col__link"><a href="{{'/blog/'|prepend:site.baseurl}}">Blog</a></div>
-        <div class="footer__cols__col__link"><a href="{{'/get-started/support/'|prepend:site.baseurl}}">Support</a></div>
+        <div class="footer__cols__col__link"><a href="{{'/community/contact-us/'|prepend:site.baseurl}}">Support</a></div>
```

Comment:
What is the reason for this change?

Reply:
Currently, if one clicks on that 'Support' link, the website takes you first to a non-existent page, and then to this page: https://beam.apache.org/community/in-person/

That page is about meetups, and not at all about support.

Reply:
btw "contact us" seems more appropriate than "support" also

Reply:
OK. That make us.

You can also rename the text of the link from Support to Contact Us, if that makes more sense here.